### PR TITLE
Fixed battle coin example and devnet sol info

### PR DESCRIPTION
**Pull request by Woody4618**
I notices the battle coin example stopped working and added new instruction on how to get dev net sol to the other examples.

### CODE REVIEW

**Review by acheroncrypto — CHANGES_REQUESTED** [review]
Thank you for the fixes but this branch needs to be updated because it doesn't have [the latest changes](https://github.com/solana-playground/solana-playground/commit/45d4ef8e012f1effaa5e63c93f1ed8766ff0ae0d#diff-ab540bd09da765512564ef46996c85490ddd68822050eab04822b89f5e1f5317L28 ) and is behind [400 commits](https://github.com/Woody4618/solana-playground/compare/update_examples...solana-playground%3Asolana-playground%3Amaster).

**Comment by Woody4618** [comment]
Oh sorry, updated

File: client/src/tutorials/BattleCoins/files/lib.rs
Comment (acheroncrypto) [inline client/src/tutorials/BattleCoins/files/lib.rs:1]: These changes should also be done in the Markdown pages https://github.com/Woody4618/solana-playground/blob/6127ce12ada7dc77f25c5229da3590ebf415d751/client/src/tutorials/BattleCoins/pages

Comment (Woody4618) [reply client/src/tutorials/BattleCoins/files/lib.rs:1]: Updated. The bumbs were already correct in the md files. Do you already know when you will release a new playground version?

Comment (acheroncrypto) [reply client/src/tutorials/BattleCoins/files/lib.rs:1]: > Updated.

I think we also need the `MetadataAccount::find_pda` change:

https://github.com/solana-playground/solana-playground/pull/181/files#diff-ab540bd09da765512564ef46996c85490ddd68822050eab04822b89f5e1f5317L162-R165

> The bumbs were already correct in the md files.

Bumps were also correct in also Rust files, updated in https://github.com/solana-playground/solana-playground/commit/45d4ef8e012f1effaa5e63c93f1ed8766ff0ae0d but not yet published.

> Do you already know when you will release a new playground version?

Most likely later this month.

File: client/src/tutorials/TinyAdventureTwo/files/lib.rs
Comment (acheroncrypto) [inline client/src/tutorials/TinyAdventureTwo/files/lib.rs:1]: This tutorial also needs code sync for Markdown files.

Comment (acheroncrypto) [reply client/src/tutorials/TinyAdventureTwo/files/lib.rs:1]: This was marked as resolved but it's not, for example `password` parameter is removed but it exists in [Markdown side](https://github.com/Woody4618/solana-playground/blob/update_examples/client/src/tutorials/TinyAdventureTwo/pages/3.md).

**Comment by Woody4618** [comment]
Could we get this one merged? This tutorial is broken for a while now and people are complaining to me :D I merged in master and tested again. Both tutorials work fine in this branch.

**Pull request merged**